samples/meetings-notification/csharp/TargetedNotifications/Bots/TargetedInMeetingNotificationBot .cs

```diff
@@ -0,0 +1,268 @@
+﻿// <copyright file="TargetedInNotifiTargetedInNotificationMeetingBotcationMeetingBot.cs" company="Microsoft">
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// </copyright>
+
+using AdaptiveCards;
+using AdaptiveCards.Templating;
+using TargetedInMeetingNotificationBot.Models;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Teams;
+using Microsoft.Bot.Connector.Authentication;
+using Microsoft.Bot.Connector;
+using Microsoft.Bot.Schema;
+using Microsoft.Bot.Schema.Teams;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http.Headers;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web;
+
+
+namespace TargetedInMeetingNotificationBot
+{
+    public class TargetedInNotificationMeetingBot : TeamsActivityHandler
```

Comment:
I think the class name & the constructor name needs to be updated as well :)

Reply:
Yes, updated